### DISCUSSION TITLE
【Lecture 3】Circom Notes

### DISCUSSION THREAD

**Discussion by Demian101** [zkp-co-learning/zkp-co-learn#120] [Notes]
推荐食用 [0] 的 R1CS 不分及 [1] [2]，余力可看 [3]
[[0] R1CS-frontend.pdf](https://github.com/zkp-co-learning/zkp-co-learn/files/12141241/0.R1CS-frontend.pdf)
[[1] Circom Basic.pdf](https://github.com/zkp-co-learning/zkp-co-learn/files/12141198/1.Circom.Basic.pdf)
[[2] Sudoku 数独.pdf](https://github.com/zkp-co-learning/zkp-co-learn/files/12141200/2.Sudoku.pdf)
[[3] Circom - Group signature.pdf](https://github.com/zkp-co-learning/zkp-co-learn/files/12141201/3.Circom.-.Group.signature.pdf)

**Comment by 0xqige** [comment]
“不分” -> "部分"

**Reply by ShirleneLiu** [reply]
hhhhh
